dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/core/database.py

```diff
@@ -52,5 +55,10 @@ def get_database_info(self, name) -> Dict:
             return self._database
         else:
             gateway = launch_gateway()
-            self._database = gateway.entry_point.getDatasourceInfo(name)
+            try:
+                self._database = gateway.entry_point.getDatasourceInfo(name)
+            # Handler database source do not exists error, for now we just terminate the process.
+            except Py4JJavaError:
```

Comment:
Except error is not just a situation. Should we use `Py4JJavaError.java_exception` to display clear information?

Reply:
It seems that `Py4JJavaError` only has one single subclass `java_exception`. https://www.py4j.org/py4j_java_protocol.html#py4j.protocol.Py4JJavaError. And it more like we could get detail java gateway information from client-side. But I would try it in my local environment.